### PR TITLE
Add new document to talk about extra sphinx tips

### PR DESCRIPTION
**Pull request by TibsAtWork**
Provide a place for sphinx/reStructuredText information that doesn't belong in README or CONTRIBUTING

See

* https://github.com/aiven/devportal/blob/tibs-sphinx-tips/SPHINX-TIPS.rst
* https://github.com/aiven/devportal/blob/tibs-sphinx-tips/README.rst
* https://github.com/aiven/devportal/blob/tibs-sphinx-tips/CONTRIBUTING.rst

Suggesting a different name of the new file, or deciding the text belongs in an existing file instead, is definitely in scope for this PR.

Fix: https://github.com/aiven/devportal/issues/712

### CODE REVIEW

**Comment by lornajane** [comment]
How about sharing the documentation about our documentation as part of our documentation? I know you like documentation!

(I'd also like to discuss/review some of these new tips) I think using the existing platform could make content more findable rather than in the buckets of randomness as it is right now, README and CONTRIBUTING are already fairly epic, we're adding these tips plus additional contribution approaches .... let's use something bookmarkable/linkable/navigable and nice to look at

**Comment by ftisiot** [comment]
@TibsAtWork are you going to work on Lorna's comment?

**Comment by TibsAtWork** [comment]
I'm happy if you want to take it on, @ftisiot - I don't think I'd get round to it in the next couple of days.

**Comment by ftisiot** [comment]
Following @lornajane's comment, I put the Tips&tricks in the same format as our devportal under a "Documentation" section.
Could you please review and comment, I feel this is far from perfect

**Comment by laysauchoa** [comment]
I thin this is also already good. However, we discussed the redirection today and I wonder if we can make a note not encouraging contributors to make redirection only if really needed it. I am not sure what would be the criteria here.

File: docs/documentation/tips-tricks/page-redirects.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Handle page redirects
```
Comment (lornajane) [inline docs/documentation/tips-tricks/page-redirects.rst:1]: Could we please remove this page? I'm opening #882 to take a less hacky approach to this problem by maintaining a list of redirects in configuration in the project.

File: docs/documentation/tips-tricks/anonymus-links.rst [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Create anonymous links
```
Comment (lornajane) [inline docs/documentation/tips-tricks/anonymus-links.rst:1]: File name is mispelled

File: docs/documentation/tips-tricks/page-redirects.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    .. raw:: html
+
+        <script type="text/javascript">
+            window.location.replace('/docs/platform/concepts/service-forking.html');
+        </script>
+
+    This page is a redirect from ``database-forking`` to ``service-forking``, since the original name (``database-forking``) was incorrect.
+    It's part of `PR #710 <https://github.com/aiven/devportal/pull/710>`_
+
+The ``:orphan:`` section tells Sphinx not to include this page in any contents list, which means that it does not issue a warning that ``database-forking.rst`` isn't included in the ``_toc.yaml`` file.
```
Comment (lornajane) [inline docs/documentation/tips-tricks/page-redirects.rst:32]: This orphan thing should be a tip by itself, we may want to have pages that aren't in the nav

**Review by lornajane — CHANGES_REQUESTED** [review]
This redirects thing makes me really nervous, let's remove it from here and try to solve it a different way.

**Review by lornajane — APPROVED** [review]

**Pull request merged**